plugins/thunar-apr/thunar-apr-image-page.c

```diff
@@ -173,7 +172,9 @@ thunar_apr_image_page_init (ThunarAprImagePage *image_page)
   /* add labels for the Exif info */
   for (n = 0; n < G_N_ELEMENTS (TAIP_EXIF); ++n)
     {
-      label = gtk_label_new (_(TAIP_EXIF[n].name));
+      const gchar *exif_title = exif_tag_get_title ( TAIP_EXIF[n] );
```

Comment:
1. OK, I'm not familiar with the lib, maybe using deprecated call makes sense here.

2. The translations list seems to be way shorter than Thunar's... not sure if we want to sacrifice existing translations in favor of upstream ones.
Also, if there's a bug/improvement for a translation, user's natural move would be to submit it against Thunar, not libexif.